composer.json

```diff
@@ -39,44 +39,44 @@
         "pagerfanta/pagerfanta": "^1.0.4",
         "piwik/device-detector": "^3.7",
         "ramsey/uuid": "^3.1",
-        "sensio/framework-extra-bundle": "^3.0",
+        "sensio/framework-extra-bundle": "^5.0",
         "stof/doctrine-extensions-bundle": "^1.2.2",
         "sulu/document-manager": "~0.10.1",
         "symfony-cmf/routing-bundle": "^2.0",
-        "symfony/asset": "^3.0",
-        "symfony/cache": "^3.0",
-        "symfony/config": "^3.0",
-        "symfony/yaml": "^3.0",
-        "symfony/console": "^3.0",
-        "symfony/dependency-injection": "^3.0",
-        "symfony/doctrine-bridge": "^3.0",
-        "symfony/event-dispatcher": "^3.0",
-        "symfony/expression-language": "^3.0",
-        "symfony/filesystem": "^3.0",
-        "symfony/finder": "^3.0",
-        "symfony/form": "^3.0",
-        "symfony/framework-bundle": "^3.0",
-        "symfony/http-foundation": "^3.0",
-        "symfony/http-kernel": "^3.0",
-        "symfony/intl": "^3.0",
-        "symfony/options-resolver": "^3.0",
+        "symfony/asset": "^3.4",
+        "symfony/cache": "^3.4",
+        "symfony/config": "^3.4",
+        "symfony/yaml": "^3.4",
+        "symfony/console": "^3.4",
+        "symfony/dependency-injection": "^3.4",
+        "symfony/doctrine-bridge": "^3.4",
+        "symfony/event-dispatcher": "^3.4",
+        "symfony/expression-language": "^3.4",
+        "symfony/filesystem": "^3.4",
+        "symfony/finder": "^3.4",
+        "symfony/form": "^3.4",
+        "symfony/framework-bundle": "^3.4",
+        "symfony/http-foundation": "^3.4",
+        "symfony/http-kernel": "^3.4",
+        "symfony/intl": "^3.4",
+        "symfony/options-resolver": "^3.4",
         "symfony/polyfill-mbstring": "~1.0",
         "symfony/polyfill-php72": "~1.0",
-        "symfony/property-access": "^3.0",
-        "symfony/proxy-manager-bridge": "^3.0",
-        "symfony/routing": "^3.0",
-        "symfony/security": "^3.0",
-        "symfony/security-bundle": "^3.0",
-        "symfony/swiftmailer-bundle": "^3.0",
-        "symfony/templating": "^3.0",
-        "symfony/translation": "^3.0",
-        "symfony/twig-bridge": "^3.0",
-        "symfony/twig-bundle": "^3.0",
-        "symfony/validator": "^3.0",
-        "symfony/web-link": "^3.0",
+        "symfony/property-access": "^3.4",
+        "symfony/proxy-manager-bridge": "^3.4",
+        "symfony/routing": "^3.4",
+        "symfony/security": "^3.4",
+        "symfony/security-bundle": "^3.4",
+        "symfony/swiftmailer-bundle": "^2.6.4",
+        "symfony/templating": "^3.4",
+        "symfony/translation": "^3.4",
+        "symfony/twig-bridge": "^3.4",
+        "symfony/twig-bundle": "^3.4",
+        "symfony/validator": "^3.4",
+        "symfony/web-link": "^3.4",
         "twig/twig": "^1.11 || ^2.0",
         "willdurand/hateoas-bundle": "^1.1",
-        "sensio/distribution-bundle": "~5.0.6"
+        "sensio/distribution-bundle": "~5.0.19"
```

Comment:
updated by requirement of https://github.com/symfony/symfony-standard/blob/v3.4.6/composer.json#L21